configure.ac

```diff
@@ -347,15 +348,15 @@ AC_CHECK_LIB([expect], [exp_expectl], [AM_CONDITIONAL([HAVE_EXPECT], [true])],
     [AC_MSG_NOTICE([libexpect not found, will not be able to run functional tests])])
 
 ## Default parameters
-AM_CFLAGS="-Wall -Wno-deprecated-declarations -std=gnu99"
+AM_CFLAGS="$AM_CFLAGS -Wall -Wno-deprecated-declarations -std=gnu99"
```

Comment:
Shouldn't this be appended? It's the user who defines it, so it should take precedence over our defaults.